source/Nuke.Common/Attributes/LatestMavenVersionAttribute.cs

```diff
@@ -29,9 +29,9 @@ public LatestMavenVersionAttribute(string repository, string groupId, string art
         public override object GetValue(MemberInfo member, object instance)
         {
             var endpoint = _repository.TrimStart("https").TrimStart("http").TrimStart("://").TrimEnd("/");
-            var uri = $"https://{endpoint}/m2/{_groupId.Replace(".", "/")}/{_artifactId ?? _groupId}/maven-metadata.xml";
+            var uri = $"https://{endpoint}/{_groupId.Replace(".", "/")}/{_artifactId ?? _groupId}/maven-metadata.xml";
             var content = HttpTasks.HttpDownloadString(uri);
-            return XmlTasks.XmlPeekFromString(content, ".//version").Last();
+            return XmlTasks.XmlPeekFromString(content, ".//latest").Single();
```

Comment:
Sorry, we need to discuss this change. Why exactly was it needed? Aren't the versions sorted?

In one of my projects I added a custom duplicate of the attribute with a new property `IncludePrereleases`. With that I'm selecting the version like so:

```
XmlTasks.XmlPeek(rssFile, ".//version").Last(x => !x.Contains('-') || IncludePrerelease);
```

Reply:
It is indeed as you mentioned, depending on the repository... the versions aren't sorted *consistently*.

Observe the following 2 metadata XML files, while also taking in mind that the current latest version (1.8.10) wasn't available/released at the time I created/"fixed" this issue.
https://plugins.gradle.org/m2/org/jetbrains/kotlin/jvm/org.jetbrains.kotlin.jvm.gradle.plugin/maven-metadata.xml
and
https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/jvm/org.jetbrains.kotlin.jvm.gradle.plugin/maven-metadata.xml

Anyways, with the previous implementation, it would return this.
```
╬═══════════════════════
║ PrintMavenVersions
╬══════════════
​
05:08:21 [INF] Latest Gradle Maven Version: 1.8.0-RC2
05:08:21 [INF] Latest Apache Maven Version: 1.8.0
```
While both repositories have knowledge of both the RC versions, as well as the stable one, they would return different "latest" versions, hence why I switched to the latest property instead. I hope this clarifies the change.

Reply:
How about this? https://github.com/nuke-build/nuke/commit/79cb086f32a41ce62c023c5ffc5e1009717405e4

Reply:
Looks good generally speaking, however I have a 2 small remarks/questions.
1) Does `NuGetVersion.Parse` throw an exception when it can't parse a version or not?
2) I can't help but feel like using `NuGetVersion` feels _off_ when the intent is to try and parse Maven versions.

Reply:
What's the difference between Maven and NuGet versions?

Reply:
I'm currently not really aware of any differences between Maven and Nuget versioning (not that I have much experience with Maven anyways).
That being said, I know that the second point is very nit-picky and just wanted to mention it as its naming *implies* that it is not supposed to be used for things other than NuGet, which might result in confusion in future.
It was however not with the intention that there had to be acted upon 🙂